helm/prometheus-rules/templates/alerting-rules/kyverno.all.rules.yml

```diff
@@ -0,0 +1,76 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: kyverno.all.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: webhooks
+    rules:
+    - alert: KyvernoWebhooksPresent
+      annotations:
+        description: 'Kyverno has no available replicas but webhooks are present.'
+        opsrecipe: kyverno-webhooks/
+      expr: sum(kube_validatingwebhookconfiguration_info{validatingwebhookconfiguration=~"kyverno-.*"}) > 0 and sum(kube_deployment_status_replicas{deployment=~"kyverno|kyverno-admission-controller"}) == 0
```

Comment:
Conscious decision: let's wait and see if we hit a case where we need to also monitor `mutatingwebhookconfigurations`. I expect that cases where there's a mutating webhook and no validating webhook are virtually nonexistent